### ISSUE TITLE
Ejectify is not working

### ISSUE THREAD

**Issue by boxerfan89** [nielsmouthaan/ejectify-macos#67]
Hi, after launching ejectify I get the following entry in launchd.log, no eject working of course. Any ideas?
cheers Heinz
2024-01-17 16:38:52.018815 (gui/501/nl.nielsmouthaan.Ejectify-LaunchAtLoginHelper [94156]) <Notice>: xpcproxy spawned with pid 94156
2024-01-17 16:38:52.018865 (gui/501/nl.nielsmouthaan.Ejectify-LaunchAtLoginHelper [94156]) <Notice>: internal event: SPAWNED, code = 0
2024-01-17 16:38:52.018878 (gui/501/nl.nielsmouthaan.Ejectify-LaunchAtLoginHelper [94156]) <Notice>: service state: xpcproxy
2024-01-17 16:38:52.018892 (gui/501/nl.nielsmouthaan.Ejectify-LaunchAtLoginHelper [94156]) <Notice>: deferred event: domain spawn response: 0
2024-01-17 16:38:52.018908 (gui/501/nl.nielsmouthaan.Ejectify-LaunchAtLoginHelper [94156]) <Notice>: internal event: SOURCE_ATTACH, code = 0
2024-01-17 16:38:52.027218 (gui/501 [100011]) <Warning>: denied lookup: name = com.apple.audio.audiohald, flags = 0x9, requestor = Google Chrome H[1718], error = 159: Sandbox restriction
2024-01-17 16:38:52.027617 (gui/501 [100011]) <Warning>: denied lookup: name = com.apple.audio.audiohald, flags = 0x9, requestor = Code Helper (GP[1729], error = 159: Sandbox restriction
2024-01-17 16:38:52.051538 (gui/501/nl.nielsmouthaan.Ejectify-LaunchAtLoginHelper [94156]) <Error>: Service could not initialize: posix_spawn((null)), error 0xe - Bad address
2024-01-17 16:38:52.051564 (gui/501/nl.nielsmouthaan.Ejectify-LaunchAtLoginHelper [94156]) <Error>: initialization failure: 23C71: xpcproxy + 31472 [1097][6960F486-3261-3A05-9150-1B1F72E3ADB0]: 0xe
2024-01-17 16:38:52.051569 (gui/501/nl.nielsmouthaan.Ejectify-LaunchAtLoginHelper [94156]) <Notice>: internal event: INIT, code = 14
2024-01-17 16:38:52.051575 (gui/501/nl.nielsmouthaan.Ejectify-LaunchAtLoginHelper [94156]) <Notice>: job state = spawn failed
2024-01-17 16:38:52.052521 (gui/501/nl.nielsmouthaan.Ejectify-LaunchAtLoginHelper [94156]) <Notice>: xpcproxy exited due to exit(78)
2024-01-17 16:38:52.052527 (gui/501/nl.nielsmouthaan.Ejectify-LaunchAtLoginHelper [94156]) <Notice>: exited due to exit(78)
2024-01-17 16:38:52.052531 (gui/501/nl.nielsmouthaan.Ejectify-LaunchAtLoginHelper [94156]) <Notice>: already handled failed init, ignoring
2024-01-17 16:38:52.052535 (gui/501/nl.nielsmouthaan.Ejectify-LaunchAtLoginHelper [94156]) <Notice>: service state: exited
2024-01-17 16:38:52.052544 (gui/501/nl.nielsmouthaan.Ejectify-LaunchAtLoginHelper [94156]) <Notice>: internal event: EXITED, code = 0
2024-01-17 16:38:52.052550 (gui/501 [100011]) <Notice>: service inactive: nl.nielsmouthaan.Ejectify-LaunchAtLoginHelper
2024-01-17 16:38:52.052577 (gui/501/nl.nielsmouthaan.Ejectify-LaunchAtLoginHelper [94156]) <Notice>: service state: not running
2024-01-17 16:38:52.052586 (gui/501/nl.nielsmouthaan.Ejectify-LaunchAtLoginHelper) <Notice>: Service only ran for 0 seconds. Pushing respawn out by 10 seconds.
2024-01-17 16:38:52.052722 (gui/501/nl.nielsmouthaan.Ejectify-LaunchAtLoginHelper) <Notice>: internal event: WILL_SPAWN, code = 0
2024-01-17 16:38:52.052754 (gui/501/nl.nielsmouthaan.Ejectify-LaunchAtLoginHelper) <Notice>: service state: spawn scheduled
2024-01-17 16:38:52.052758 (gui/501/nl.nielsmouthaan.Ejectify-LaunchAtLoginHelper) <Notice>: service throttled by 10 seconds

**Comment by nielsmouthaan** [comment]
Can you see the app running in the status bar? Please also check [the FAQ](https://github.com/nielsmouthaan/ejectify-macos/blob/main/README.md#frequently-asked-questions) to see if it contains anything that can help solve the issue you're experiencing.

**Comment by boxerfan89** [comment]
Yes, app is showing up in status bar. Already checked FAQ, nothing helped. No difference between admin user of not. Anything else I can try?

**Comment by nielsmouthaan** [comment]
What exactly is the issue you're experiencing?

**Comment by boxerfan89** [comment]
Well, the app does not unmount the external drive. I tried all options in the app, different usb ports. No change, still this annoying messages.

**Comment by nielsmouthaan** [comment]
Have you tried the force unmount option, too? That surely should unmount the drive as it doesn't wait for other processes to stop using the volume.

**Comment by boxerfan89** [comment]
yes, but no reaction. If I use "unmount all" however, the drives are unmounted.

**Comment by nielsmouthaan** [comment]
Then there's either a process that stops Ejectify from unmounting the volumes while your Mac starts sleeping (or whatever trigger you've configured) or the trigger isn't executed (e.g. set to eject when your Mac starts sleeping but your Mac actually doesn't sleep, but its displayed are just turned off).

**Comment by boxerfan89** [comment]
well, my Mac clearly starts sleeping, otherwise the messages won´t appear. Any idea how I could identify the process which is stopping ejectify?

**Comment by nielsmouthaan** [comment]
See [this FAQ](https://github.com/nielsmouthaan/ejectify-macos#why-do-i-still-receive-notifications), especially the 4th bullet point.

**Comment by nielsmouthaan** [comment]
I've seen these kinds of cards cause issues, indeed. Might be worth checking how it works when connecting them directly.
